README.md

```diff
@@ -1,42 +1,31 @@
 ![Project Tox](https://raw.github.com/TokTok/toxcore/master/other/tox.png "Project Tox")
 ***
 
-**Current build status:** [![Build Status](https://travis-ci.org/TokTok/toxcore.svg?branch=master)](https://travis-ci.org/TokTok/toxcore)
+**Current build status:** [![Build Status](https://travis-ci.org/TokTok/c-toxcore.svg?branch=master)](https://travis-ci.org/TokTok/c-toxcore)
 **Current Coverage:** [![Coverage Status](https://coveralls.io/repos/github/TokTok/toxcore/badge.svg?branch=master)](https://coveralls.io/github/TokTok/toxcore?branch=master)
 
 [**Website**](https://tox.chat) **|** [**Wiki**](https://wiki.tox.chat/) **|** [**Blog**](https://blog.tox.chat/) **|** [**FAQ**](https://wiki.tox.chat/doku.php?id=users:faq) **|** [**Binaries/Downloads**](https://wiki.tox.chat/Binaries) **|** [**Clients**](https://wiki.tox.chat/doku.php?id=clients) **|** [**Compiling**](/INSTALL.md)
 
-**IRC Channels:** [#tox@freenode](https://webchat.freenode.net/?channels=tox), [#toktok@freenode](https://webchat.freenode.net/?channels=toktok)
+**IRC Channels:** Users: [#tox@freenode](https://webchat.freenode.net/?channels=tox), Developers: [#toktok@freenode](https://webchat.freenode.net/?channels=toktok)
 
-## Q&A:
-
-### What is Tox?
-
-Tox is a fully encrypted, censor resistant, private, distributed network library with a focus on personal communications.
-
-### No, really, what's Tox?
-
-It's a VERY secure Instant Messenger that supports Text, Audio/Video calls, group chats, audio group chats, and file transfers.
-
-### What are your goals with Tox?
-
-We want Tox to be as simple as possible while remaining as secure as possible.
-
-## Toxcore Roadmap
+## Toxcore Development Roadmap
 This Roadmap is somewhat tentative, but should give you a good idea of where
 we're going, and where we've been.
 
 Currently unsorted, the following is intended to function as a discussion guide
 to developers/contributors.
 
 ### In Progress
-- [ ] 100% unit testing
-- [ ] Make toxcore stateless
-- [ ] Allow a single toxcore instance to handle multiple keypairs
-- [ ] Implement usable group chats
-- [ ] Improve A/V implementation
-- [ ] Multiple device support
-- [ ] Consistent naming scheme throughout toxcore
+- [ ] Toxcore
+    - [ ] 100% unit testing
+    - [ ] Make ToxAV stateless
+    - [ ] Allow a single toxcore instance to handle multiple keypairs (or 'clients')
+    - [ ] Consistent naming scheme throughout toxcore
+    - [X] Make toxcore stateless
+- [ ] Messenger
+    - [ ] Improve group chat implementation
+    - [ ] Improve A/V implementation
```

Comment:
Because the former (in my head) is changing the public API, giving more control to the clients. Whereas making it stateless allows us to remove the thread_mutexes from a deeper level.

Inprovments to ToxAV could be solely in ToxAV (which sits on top of messenger), making it stateless requires you to know about ToxAV and Toxcore.

I don't know how to make `Messenger` more clear, I'm open to suggestions.